### PR TITLE
More stubs and header additions from #768 for systemd

### PR DESCRIPTION
**Pull request by Dennisbonke**
Mostly empty headers, a few defines and 2 function stubs. Part 3 of the systemd master PR.

Part of the systemd on Managarm project.

### CODE REVIEW

**Review by no92 — APPROVED** [review]
LGTM

**Comment by Dennisbonke** [comment]
Rebased and dropped the linux-headers.

File: options/linux/include/sys/mount.h [hunk 1]
```diff
@@ -15,6 +15,10 @@ extern "C" {
 #define MS_PRIVATE 128
```
Comment (no92) [inline options/linux/include/sys/mount.h:15]: This definition conflicts with the new definition 3 lines below.

**Review by Geertiebear — APPROVED** [review]
LGTM

**Pull request merged**